### DISCUSSION TITLE
`SelectElement`: how to re-fetch items with async function when other field changes?

### DISCUSSION THREAD

**Discussion by letmaik** [vueform/vueform#213] [Questions] [answered]
https://vueform.com/reference/select-element#async

The example shows how to fetch items based on the value of another field. But what's not clear is how the items are re-fetched when the other field changes. It seems like there's some logic missing.

```vue
<SelectElement :native="false" :items="async function(query, input){
  let el$ = input.$parent.el$ // the element's component
  let other_element = el$.form$.el$('other_element') // an other element

  return await (axios.get('https://vueform.com/json/async-items.json')).data
}" ... />
```

**Comment by adamberecz** [comment, marked as answer]
Here here:
https://vueform.com/recipes/dynamically-load-options-to-chained-select-boxes-with-async-functions
